### ISSUE TITLE
One little question

### ISSUE THREAD

**Issue by Erhushenshou** [jlongLondon/CT-Dose#1]
You seemed to export certain slice of CT and its corresponding dose map. But the actual situation is that we want the CT plane image and isodoseline imposed on it. How could we figure out this to transfer dosemap to isodoseline?

**Comment by Jonathan-mark-long** [comment]
Yeah, I am really not sure. You would have to find a way of getting the dose value of each isodose line you want to be shown. You could then iterate through the dosemap searching for each value. From here you could replace the pixel in the CT image, at the coordinates of the found value in the dosemap, with the colour of the isodose line. Alternatively the Isodose Class provides a MeshGeometry, so a potential might be the System.Windows.Shapes.Polygon class. I have unfortunately not looked into either, and this was the point I gave up in the past. Sorry I can't be of more help!

**Comment by Erhushenshou** [comment]
Many thanks to your hints. Do consider to draw spots in proton plan?

**Comment by Erhushenshou** [comment]
I do implement to draw the transverse plane isodoseline. But for sagital or coronal plane, the method which only supports transversal dose or image voxel export prevent from building the sagital or coronal plane quickly.

**Comment by Jonathan-mark-long** [comment]
Hey,

Unfortunately I have not looked into proton planning. 

To answer your second question I have created a new branch called Adding_Sagital_Coronal. This branch shows you how you could create a sagital and coronal dose map. I did a quick test and images that looked right at a quick glance were created, there is more than likely some issues with rotation though as I put this together quite quickly. So you may need to switch X/Y/Z around to get the correct orientation, any questions let me know. 

I would be very interested in seeing how you are drawing the isodoseline if this was something you wanted to share with me.

Good Luck,
John
